Description: Fix spelling errors.
 calulated       -> calculated
 correspoding    -> corresponding
 specifed        -> specified
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://trac.osgeo.org/grass/ticket/2985
Applied-Upstream: https://trac.osgeo.org/grass/changeset/68236

--- a/raster/r.solute.transport/r.solute.transport.html
+++ b/raster/r.solute.transport/r.solute.transport.html
@@ -60,7 +60,7 @@ The calculation and boundary status of s
 The following states are supportet:
 
 <ul>
-<li>0 == inactive - the cell with status 0 will not be calulated, active cells will have a no flow boundary to an inactive cell</li>
+<li>0 == inactive - the cell with status 0 will not be calculated, active cells will have a no flow boundary to an inactive cell</li>
 <li>1 == active - this cell is used for sloute transport calculation, inner sources can be defined for those cells</li>
 <li>2 == Dirichlet - cells of this type will have a fixed concentration value which do not change over time </li>
 <li>3 == Transmission - cells of this type should be placed on out-flow boundaries to assure the flow of the solute stream out </li>
--- a/raster3d/r3.gwflow/r3.gwflow.html
+++ b/raster3d/r3.gwflow/r3.gwflow.html
@@ -47,7 +47,7 @@ status of single cells can be set with t
 cell states are supported:
 
 <ul>
-<li>0 == inactive - the cell with status 0 will not be calulated,
+<li>0 == inactive - the cell with status 0 will not be calculated,
  active cells will have a no flow boundary to an inactive cell</li>
 <li>1 == active - this cell is used for groundwater calculation,
  inner sources can be defined for those cells</li>
--- a/temporal/t.rast.algebra/t.rast.algebra.html
+++ b/temporal/t.rast.algebra/t.rast.algebra.html
@@ -246,7 +246,7 @@ compared by logical operators and evalua
 the related maps.
 For complex relations the comparison operator can be used to combine conditions: 
 <br>
-The structure is similar to the select operator with the extention of an aggregation operator: 
+The structure is similar to the select operator with the extension of an aggregation operator: 
 {"comparison operator", "topological relations", aggregation operator, "temporal operator"}
 <br>
 This aggregation operator (| or &) define the behaviour if a map is related the more 
--- a/vector/v.out.ogr/export_areas.c
+++ b/vector/v.out.ogr/export_areas.c
@@ -77,7 +77,7 @@ int export_areas_single(struct Map_info
                        * not labeled */
         }
         
-        /* find correspoding area */
+        /* find corresponding area */
         area = Vect_get_centroid_area(In, centroid);
         if (area == 0)
             continue;
@@ -184,7 +184,7 @@ int export_areas_multi(struct Map_info *
                                 "Category %d not found in line %d, field %d"),
                               cat, line, field);
             
-            /* find correspoding area */
+            /* find corresponding area */
             area = Vect_get_centroid_area(In, line);
             if (area == 0)
                 continue;
@@ -241,7 +241,7 @@ int export_areas_multi(struct Map_info *
                        * not labeled */
         }
 
-        /* find correspoding area */
+        /* find corresponding area */
 	line = Vect_get_next_line_id(In);
         area = Vect_get_centroid_area(In, line);
         if (area == 0)
--- a/general/g.proj/datumtrans.c
+++ b/general/g.proj/datumtrans.c
@@ -91,7 +91,7 @@ int set_datum(char *datum)
  *        co-ordinate system definition
  * 
  * \param datumtrans  Index number of parameter set to use, 0 to leave
- *                    unspecifed (or remove specific parameters, leaving just
+ *                    unspecified (or remove specific parameters, leaving just
  *                    the datum name), -1 to list the available parameter
  *                    sets for this datum and exit.
  *
--- a/lib/proj/convert.c
+++ b/lib/proj/convert.c
@@ -264,7 +264,7 @@ OGRSpatialReferenceH GPJ_grass_to_osr(co
  * \param hSRS        OGRSpatialReferenceH object containing the co-ordinate 
  *                    system to be converted
  * \param datumtrans  Index number of datum parameter set to use, 0 to leave
- *                    unspecifed
+ *                    unspecified
  * 
  * \return            2 if a projected or lat/long co-ordinate system has been
  *                    defined
@@ -675,7 +675,7 @@ int GPJ_osr_to_grass(struct Cell_head *c
  * \param wkt         Well-known Text (WKT) description of the co-ordinate 
  *                    system to be converted
  * \param datumtrans  Index number of datum parameter set to use, 0 to leave
- *                    unspecifed
+ *                    unspecified
  * 
  * \return            2 if a projected or lat/long co-ordinate system has been
  *                    defined
